pkg/apis/sedna/v1alpha1/incrementallearningjob_types.go

```diff
@@ -95,7 +95,9 @@ type InitialModel struct {
 }
 
 type DeployModel struct {
-	Name string `json:"name"`
+	Name              string `json:"name"`
+	HotUpdateEnabled  bool   `json:"hotUpdateEnabled,omitempty"`
```

Comment:
Add a description/default value etc. for these two new fields

Reply:
fix it